qchem/tests/test_active_space.py

```diff
@@ -20,17 +22,27 @@
     ],
 )
 def test_active_spaces(
-    mol_name, n_act_electrons, n_act_orbitals, docc_indices_ref, act_indices_ref
+    mol_name, n_act_electrons, n_act_orbitals, core_ref, active_ref
 ):
 
     r"""Test the correctness of the generated active spaces"""
 
-    docc_indices, active_indices = qchem.active_space(
-        mol_name, ref_dir, n_act_electrons, n_act_orbitals
+    molecule = MolecularData(filename=os.path.join(ref_dir, mol_name))
+
+    core, active = qchem.active_space(
+        molecule.n_electrons,
+        molecule.n_orbitals,
+        mult=molecule.multiplicity,
+        nact_els=n_act_electrons,
+        nact_orbs=n_act_orbitals
```

Comment:
You might change the names `n_act_electrons` and `n_act_orbitals` simply to `nact_els` and `nact_orbs`.

Reply:
I don't like that too much. I prefer `active_electrons` and `active_orbitals`